pkg/sql/pg_catalog.go

```diff
@@ -1254,3 +1303,43 @@ func (h oidHasher) BuiltinOid(name string, builtin *parser.Builtin) *parser.DInt
 	h.writeStr(builtin.Types.String())
 	return h.getOid()
 }
+
+func (h oidHasher) UserOid(username string) *parser.DInt {
+	h.writeTypeTag(userTypeTag)
+	h.writeStr(username)
+	return h.getOid()
+}
+
+func forEachUser(p *planner, fn func(username string) error) error {
+	query := `SELECT username FROM system.users`
+	plan, err := p.query(query)
+	if err != nil {
+		return nil
+	}
+	defer plan.Close()
+	if err := plan.Start(); err != nil {
+		return err
+	}
+
+	// TODO(cuongdo/asubiotto): Get rid of root user special-casing if/when a row
```

Comment:
A couple of reasons:
1. What if the ORM-based app is running as our root user?
2. Postgres exposes the default `postgres` role in `pg_roles`
3. It seems less surprising that `root`, a user that exists as far as a normal CockroachDB is concerned, is in `pg_roles`